dax/controller/client/client.go

```diff
@@ -27,15 +27,17 @@ var _ dax.Schemar = (*Client)(nil)
 // Client is an HTTP client that operates on the Controller endpoints exposed by
 // the main Controller service.
 type Client struct {
-	address dax.Address
-	logger  logger.Logger
+	address    dax.Address
+	httpClient http.Client
```

Comment:
so, if i make it a `*http.Client`, i have to initialize it on first use, and have a mutex to govern the initialization and keep from racing. and since i'm just replacing DefaultClient, I assumed that the default settings, which are used by DefaultClient, were fine. I didn't want to change the settings without a concrete reason.

Reply:
hmm. on further thought, i don't actually think i find the argument persuasive, so i think i will just copy the querier/client behavior. creating a new `http.Client` object doesn't actually consume meaningful resources.

Reply:
Either way is ok with me. I was just curious.